SwanHeader/src/header.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+
+import { Dialog, showDialog } from '@jupyterlab/apputils';
+
+import { ThemeProvider } from './theme-provider';
+
+import ButtonBase from '@material-ui/core/ButtonBase';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Avatar from '@material-ui/core/Avatar';
+
+export const Header = (props: {
+  hubPrefix?: string;
+  hubUser?: string;
+  baseUrl: string;
+}): React.ReactElement => {
+  const userName = props.hubUser || 'swanuser';
+  const shutdownUrl = (props.hubPrefix || props.baseUrl) + 'home?changeconfig';
```

Comment:
Shouldn't this always be `hubPrefix`?  Isn't `baseUrl` the base for the user container?